src/main/java/com/minecolonies/coremod/entity/ai/citizen/herders/AbstractEntityAIHerder.java

```diff
@@ -534,10 +537,18 @@ private void butcherAnimal(@Nullable final EntityAnimal animal)
      *
      * @return the BreedingItem stack.
      */
-    public ItemStack getBreedingItems()
+    public ItemStack getBreedingItems(final Boolean required)
```

Comment:
javadoc of param?

Reply:
Can we make this two methods? One is getBreedingItems() and one is requestBreedingItem() (or so?)

Reply:
We could... but that'd be more code. and really, i'm lazy. XD